test.py

```diff
@@ -28,18 +27,16 @@ def main():
     else:
         username = input("Username: ")
         password = input("Password: ")
-        auth = Auth(None, token_updated)
+        auth = Auth("HomeAssistant/0.105.0dev0", None, token_updated)
```

Comment:
Please do not identify as Home Assistant. I don't want people to copy the test example and take this over. 

```suggestion
        auth = Auth("MyProject/1.0", None, token_updated)
```